.github/dependabot.yml

```diff
@@ -2,28 +2,28 @@
 # https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "/"
+  - package-ecosystem: 'npm'
+    directory: '/'
```

Comment:
Is there a YAML rule that prevents all the unneeded quotations? Scalar data types in yaml are only string and numeric, but there are edge cases, of which this might be one. See: https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml

Would be nice to see some rules enforced on quotes in YAML.

Reply:
I think this is an instance of Prettier finding double quotes and replacing them with single quotes. You are correct, though, that if there were _no_ quotes, nothing would have changed. 

If we wanted to, I think we could likely get rid of quotes in a lot of these YAML files.

Reply:
> Would be nice to see some rules enforced on quotes in YAML.

@offgriddev Since we don't lint our YAML files, I don't know if there is anything I can do as part of this PR to add rules for YAML quotes. Are there any other changes you want me to make?

Reply:
@htunnicliff Sorry I missed this. You got the spirit of it in that the majority of the quotes here are unnecessary. I think we could define an `as-needed` rule for quotes on YAML in prettier, at least from what I gather after quickly looking at their docs. I'll make a card to investigate this, no action required here though.